PGPAuth/src/main/java/org/lf_net/pgpunlocker/Server.java

```diff
@@ -73,6 +97,10 @@ public JSONObject serializeJSON() {
             ret.put("name", _name);
             ret.put("url", _url);
             ret.put("apgKey", _apgKey);
+            if (saveKey()) {
```

Comment:
use private variables where possible, as function calls are slower in most languages.